qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -331,4 +331,23 @@ _sudo dd if=${DATA} of=${TEMPDIR}/mnt/test1 bs=1M count=1 oflag=direct
 _sudo umount ${TEMPDIR}/mnt
 unmap_device ${DEV} ${PID}
 
+# test map with --show-cookie and attach with --cookie parameters
+OUT=`_sudo rbd-nbd map --try-netlink ${POOL}/${IMAGE} --show-cookie`
+read -a ${DEV} <<< ${OUT} // expect "device cookie"
+get_pid
+_sudo mount ${DEV[0]} ${TEMPDIR}/mnt
+_sudo rbd-nbd detach ${POOL}/${IMAGE}
+expect_false get_pid
+_sudo rbd-nbd attach --device ${DEV[0]} ${POOL}/${IMAGE} --cookie ${DEV[1]}
```

Comment:
I suppose when running this on an old kernel, `${DEV[1]}` will be empty and the test will fail?

Could you improve this. Alternatively, may be we should allow empty '--cookie' param, for cases like this?

Reply:
oops! I would prefer to add a condition for empty string check for ${DEV[1]} and at accordingly.
Because there is `--force` option for old kernel versions now (This patch needs a refresh to use latest --force option)
But you can correct me, I don't have a strong opinion here.